src/Twig/LocaleExtension.php

```diff
@@ -426,4 +439,35 @@ private function getFlagCodes()
             'ZW' => 'Zimbabwe',
         ]);
     }
+
+    public function localedatetime($dateTime, string $format = '%B %e, %Y %H:%M', ?string $locale = '0')
```

Comment:
return type

Reply:
add phpdoc what is accepted as $dateTime

Reply:
done.